### PR TITLE
more odds and ends

### LINKED ISSUE #140: include PA14 homologue number for genes?
*From Georgia Doing:*

Could we include PA14 homologue number in the gene tables? Pa biologists would use that more than Entrez ID, but if both fit that could be nice too

---

Possibly related to #133 ?

### PR DESCRIPTION
**Pull request by vincerubinetti**
- add home page thumbnails
- add minWidth option to table component for per-table min-widths
- make table row heights constant again for aethetics, and allow very wide tables (via min width) to show big cells without truncating them
- change table to always add nowrap span to cells if element rendered isn't react component
- put automatic rounding of numbers right into table to reduce repetition
- include gene aliases field in gene search, closing #140 
- remove duplicate `entrezid` field (duplicate of `entrezId`)
- rename help page "about"
- change about page content a bit
- add "about models" information, copied from old adage and touched up a bit
- remove "about the app section". using the app should require no explanation
- put shortcut keys for sections as instructions in section header tooltip
- tweak homepage visualization again
- css tweaks

### CODE REVIEW

**Comment by vincerubinetti** [comment]
@dongbohu Was going to have Casey review this text content of this PR, but since I don't think he'll have time for a while, and since this PR covers a lot more than just the text content, I've created #161 instead. Casey and others can review the text, right in the deployed app (a little easier than a github diff), and give me feedback in that issue.

File: src/pages/about/models.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    <p className='help_header weight_semibold size_medium'>
+      Expression compendium
+    </p>
+    <p>
+      The models make use of the Pseudomonas Aeruginosa expression compendium.
+      It was built on July 31 2015 and contains 125 datasets with 1,051
+      individual samples. Each dataset is an experiment and all samples are
+      collected in a non-redundant way into the compendium. If a sample is part
+      of multiple experiments, it is only represented once in the compendium. It
+      did not include any data sets uploaded after the build date. It was
```
Comment (dongbohu) [inline src/pages/about/models.js]: "data sets" to "datasets"?

File: src/pages/about/models.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      as the authors chose. Experiments vary in the number of samples they
+      contain. Samples may be from different conditions or replicates. The
+      annotations that accompany each experiment can help interpret the samples
+      and signature activities within each experiment.
+    </p>
+    <p className='help_header weight_semibold size_medium'>Samples</p>
+    <p>
+      We include samples measured on the Affymetrix Pseudomonas aeruginosa Array
+      "Pae_G1a" (labeled as A-AFFY-30 in ArrayExpress database). Built from
+      accessible data sets from the{' '}
```
Comment (dongbohu) [inline src/pages/about/models.js]: "datasets"?

**Review by dongbohu — COMMENTED** [review]
Found two minor issues with the text. Need more careful reviews from other lab members (as addressed in issue #161). Feel free to close it after addressing the two comments that I made.

**Review by dongbohu — APPROVED** [review]

**Pull request merged**